core/datainput.gms

```diff
@@ -428,14 +428,25 @@ pm_cf(ttot,regi,te) =  f_cf(ttot,regi,te);
 *RP short-term fix: set capacity factors here by hand, because the input data procudure won't be updated in time
 pm_cf(ttot,regi,"apcardiefft") = 1;
 pm_cf(ttot,regi,"apcardieffH2t") = 1;
-pm_cf(ttot,regi,"h2turbVRE") = 0.15;
+***pm_cf(ttot,regi,"h2turbVRE") = 0.15;
```

Comment:
Could this be also added to the calcCapacityFactor function in mrremind?

This way, once new data input is created and the f_cf.cs3r file is updated this line could be simply removed from the code, and we would keep having a single place with all definitions of capacity factors assumptions.

The same would apply to the other "hard-coded" fixings that were already present in this section of the code.

Reply:
Yes, exactly. 
I currently can't find the time to do it via mrremind, so when I saw that there were still old cf corrections in the code, I went the easy route, thinking "at some point of time someone will move all these changes here to mrremind"

Reply:
o.k. for now, but I hope that this preliminary solution will not stay too long